test/linalg/special.jl

```diff
@@ -0,0 +1,105 @@
+#This file is a part of Julia. License is MIT: http://julialang.org/license
```

Comment:
This needs to start with `# This file`, the space between `#` and `This` is significant to the `add_license_to_files.jl` script.